src/lib/utils/sizes.js

```diff
@@ -72,4 +72,13 @@ const sizes = new class {
 // backward compatibility expors
 export const { getMaxDeviceWidth, getMaxDeviceHeight, getDesignRelativeWidth, getDesignRelativeHeight, measure } = sizes
 
+const width = getScreenWidth()
+const height = getScreenHeight()
+
+export const isSmallDevice = width < 350
+export const isMediumDevice = width >= 350 && width < 395
+export const isLargeDevice = width >= 395
+export const isShortDevice = height < 610
```

Comment:
Actually devices are measured by the width only. Also GoodDollar wallet uses portrait-only orientation, so in the most cases height don't matters.

I think better would be check screen ratio instead of height. And apply specific design props for 16:10 (as you called them "long") devices

Reply:
actually no. a device can be wide but short. you can not deduct from width if the whole design can fit inside screen
or you need to start shrinking components.